internal/restconfig/restconfig.go

```diff
@@ -20,6 +20,8 @@ package restconfig
 
 import (
 	"fmt"
+	"github.com/submariner-io/submariner-operator/internal/exit"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/utils"
```

Comment:
`utils` is no longer referenced.

Reply:
L302 still uses `utils.GetSubmarinerResource(config)`